internal/c8dutil/memory_label_store.go

```diff
@@ -0,0 +1,58 @@
+package c8dutil
+
+import (
+	"sync"
+
+	"github.com/opencontainers/go-digest"
+)
+
+type memoryLabelStore struct {
+	mu     sync.Mutex
+	labels map[digest.Digest]map[string]string
+}
+
+func NewMemoryLabelStore() *memoryLabelStore {
+	return &memoryLabelStore{
+		labels: make(map[digest.Digest]map[string]string),
```

Comment:
Looks like the implementation didn't change, so is still handling nil values for labels.

So you could either;


- Remove the constructor, and export `memoryLabelStore`
- Remove this line, and just return ` memoryLabelStore{}`
- Change the implementations to remove the `if s.labels == nil` checks

Reply:
Done